README.md

```diff
@@ -0,0 +1,74 @@
+# Console for Apache Kafka™ on Kubernetes
+
+This project is a web console designed to facilitate interactions with Apache Kafka™ instances on Kubernetes, leveraging the [Strimzi](https://strimzi.io) Cluster Operator. 
+It is composed of two main parts: 
+
+- a REST API backend developed with Java Quarkus
+- a user interface (UI) built with Next.js and [PatternFly](https://patternfly.org)
+
+### Getting started
+
+#### API
+
+Ensure you have Kubernetes and Strimzi Cluster Operator installed on your system.
```

Comment:
```suggestion
Ensure you have Kubernetes and Strimzi Cluster Operator installed on your system.

One option to get started is to follow [Strimzi's Quick Starts](https://strimzi.io/quickstarts/).
```

Reply:
I think we can piggyback on Strimzi's doc for this.

Reply:
totally 👍